NewSpring/corinth
# fix: Fix How to Pray Link

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR fixes the "How to Pray" link on the `PrayerSingle`.

### How do I test this PR?
Start on the `master` branch.
Run the app, go to the prayer tab, and pull up a prayer.
Tap on the "Not sure how to pray?" link. You should see the following error:
![Screen Shot 2020-09-02 at 8 36 15 AM](https://user-images.githubusercontent.com/3229463/91982734-ae632880-ecf8-11ea-91d7-800030102138.png)
Go ahead and shut that down and switch over to this `fix-prayer-link` branch.
Run the same test and you should now see the link open the correct piece of content:
![Screen Shot 2020-09-02 at 8 37 10 AM](https://user-images.githubusercontent.com/3229463/91982722-aacfa180-ecf8-11ea-95ed-b171f0403123.png)

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
